bucket/azcopy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10",
+    "version": "10.11.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://azcopyvnext.azureedge.net/release20210616/azcopy_windows_amd64_10.11.0.zip",
+            "extract_dir": "azcopy_windows_amd64_10.11.0"
+        },
+        "32bit": {
+            "url": "https://azcopyvnext.azureedge.net/release20210616/azcopy_windows_386_10.11.0.zip",
+            "extract_dir": "azcopy_windows_386_10.11.0"
+        }
+    },
+    "bin": [

```

Comment:
`bin` can be reduced to a single string if there is only 1 item.
`autoupdate` is not available, but it is possible to involve `checkver`. (and update manually when `checkver` tells us)
For details, check: https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#add-autoupdate-to-a-manifest
Or, you can just check other packages in the bucket.
```
    "bin": "azcopy.exe",
    "checkver": {
        "github": "https://github.com/Azure/azure-storage-azcopy"
    },
```